### ISSUE TITLE
Only Reading PDF Files?

### ISSUE THREAD

**Issue by ktapdasan** [skcript/cvscan#21]
Is it possible to run .doc and .docx too?

**Comment by baljeet** [comment]
You can write some code for that, i used textract instead pdfminer in my fork

**Comment by ktapdasan** [comment]
Is it possible if we can convert both PDF and .DOC / .DOCS files?

**Comment by baljeet** [comment]
Yes! but you have to write some custom code. The library I mentioned, 'textract', supports pdf, doc, docx and more.
